routers/api/v1/shared/action.go

```diff
@@ -152,6 +155,77 @@ func ListRuns(ctx *context.APIContext, ownerID, repoID int64) {
 		opts.CommitSHA = headSHA
 	}
 
+	// Handle exclude_pull_requests parameter
+	if exclude := ctx.FormString("exclude_pull_requests"); exclude != "" {
+		if exclude == "true" || exclude == "1" {
```

Comment:
Maybe `ctx.FormBool`

Reply:
Addressed, see https://github.com/go-gitea/gitea/pull/34894#issuecomment-3016900408 and https://github.com/go-gitea/gitea/pull/34894/commits/8b7ed3af421ade2adabc9896bca8d4871029b2d0